src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -27,33 +27,58 @@ def import_from_package
           end
         end
 
-        redirect_to kiwi_image_path(package.kiwi_image)
+        redirect_to edit_kiwi_image_path(package.kiwi_image, section: 'software')
```

Comment:
💭  why to edit?

Reply:
It was the show action before (and it was for showing/editing the info)... it  make more sense to edit in the edit action 😸